bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -4,6 +4,8 @@ components:
       test-configs:
         - with-security
         - without-security
+      additional-cluster-configs:
+        - "path.repo: /tmp/tmp_dir"
```

Comment:
Shouldn’t be relying on hardcoded location of temp. Maybe $TEMP and expand?

Reply:
i think we should clarify the context here why this is required. These additional configs should only be opensearch yaml settings IMO.

Reply:
Agree. Like if we put this setting in `opensearch.yml` creation and deletion of that should be taken care by the tests. Should I remove this workaround from our end and let IM plugin fix it?

Reply:
yeah, let's do that and circle back on this requirement with index-management team.